katdal/test/test_categorical.py

```diff
@@ -0,0 +1,18 @@
+"""Tests for :py:mod:`katdal.categorical`."""
+
+import numpy as np
+from numpy.testing import assert_equal
```

Comment:
Actually, `np.testing.assert_array_equal` prints both arrays. I could not get any of the other variants of `assert_equal` to do that, so switch to that instead.